src/renderer/mod.rs

```diff
@@ -148,12 +157,14 @@ impl Renderer {
         self.cursor_renderer.prepare_frame()
     }
 
-    pub fn draw_frame(&mut self, root_canvas: &Canvas, dt: f32) {
+    pub fn draw_frame(&mut self, root_canvas: &Canvas, dt: f32, width: f32, height: f32) {
```

Comment:
Might be able to use `Canvas::base_layer_size` instead of passing `width` and `height` by parameter?

(fwiw you can look up the things available in `skia_safe` on skia.org, e.g. [`SkCanvas::getBaseLayerSize`] corresponds to `Canvas::base_layer_size`. The docs for `skia_safe` are available directly by running `cargo doc --open` locally)

Reply:
With the latest renderer changes using `base_layer_size` makes things much more cleaner however it doesn't cover full screen. There is some space left in the bottom that isn't covered by the image. 
<img width="579" alt="image" src="https://github.com/neovide/neovide/assets/12953777/b26bf3f3-e347-4b70-88c0-0b70d3433cd1">
I don't think it's very critical but it would be nice if image could cover the whole screen. Do you know how this can be achived?